benchmarks/bench_20newsgroups.py

```diff
@@ -59,7 +58,7 @@
     print("X_test.format = {0}".format(X_test.format))
     print("X_test.dtype = {0}".format(X_test.dtype))
     print("y_test {0}".format(y_test.shape))
-    print()
```

Comment:
These changes are irrelevant to the PR. Please only include changes that are related.